modules/web/client/src/main/scala/observe/ui/components/queue/SessionQueue.scala

```diff
@@ -219,8 +221,8 @@ object SessionQueue:
             tableMod = ObserveStyles.ObserveTable |+| ObserveStyles.SessionTable,
             rowMod = row =>
               TagMod(
-                rowClass( /*row.index.toInt,*/ row.original, props.selectedObsId.get),
-                ^.onClick --> props.selectedObsId.set(row.original.obsId.some)
+                rowClass( /*row.index.toInt,*/ row.original, props.selectedObsId),
+                ^.onClick --> props.setSelectedObsId(row.original.obsId)
```

Comment:
load on click then? what if it is running?

Reply:
I will switch to a load button in a future PR.